src/Fixer/CastNotation/ModernizeTypesCastingFixer.php

```diff
@@ -138,6 +145,10 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                     $tokens->removeTrailingWhitespace($functionName);
                 }
 
+                if ($wrapInParenthesises) {
+                    $tokens->insertAt($closeParenthesis, new Token(')'));
```

Comment:
Thanks for the reactivity, but what will this result for e.g. `<?php $foo = strval($x + $y)[0];`? What about combining with the existing `$preserveParenthesises`?

Reply:
OK I just tried your fix and the result is `<?php $foo = ((string) ($x + $y))[0];`, which is good! :+1: might be worth to add this case to the tests ;)